### DISCUSSION TITLE
Target #134 top solution rn is 93 chars.... BUT HOW???

### DISCUSSION THREAD

**Discussion by joewinbpaul** [cssbattle/forum#333] [General]
In target#134 (Wig)
![134](https://user-images.githubusercontent.com/102139966/207308263-24c68de9-467e-4896-99d7-3efa568d49d4.png)
, My smallest code consists of 120 chars but the top solution is literally  93 chars.. <p> Is it done by unicode or just pure css?? <p>` _Or is it some sort of cheating....._`

**Comment by SelenIT** [comment]
AFAIK, it's unicode, like most of the extremely short solutions. However, it's possible to do this target in 115 chars with no unicode (using one non-standard `-webkit-`prefixed css property)

**Reply by joewinbpaul** [reply]
I've been experimenting with the code a lot and the best I got using unicode is 143 chars. <p> I used the ✦ unicode <p> Is it the right Unicode character?

**Reply by joewinbpaul** [reply]
I got a 120 char solution _without Unicode_ using `box-reflect`

**Reply by SelenIT** [reply]
Yes, it's the property I meant ;)

**Reply by joewinbpaul** [reply]
> Yes, it's the property I meant ;)

But I only got a minimum of 120 chars. How could u get 115 chars with that?
Did u use some other 'special' property with it?

**Reply by SelenIT** [reply]
One property can provide both the color and the minimum width of the yellow part, and one of the font-relative units helps shortening its numeric part to just 1 digit. The margin can have just 2 values with the better value of box-reflect. And the ancient attribute of the `body` tag allows to shorten the background color (#62306D can be "approximated" with #623070, and trailing zeroes in that attribute can be omitted)

**Reply by joewinbpaul** [reply]
... I only understood the last clue of the `body` tag. <p> I am kinda confused with how the margin uses just 2 values with the  _"better value of box-reflect"_.  <p> But I'll just keep experimenting to understand what u meant.....

**Reply by joewinbpaul** [reply]
This was the code I used with unicode : <p> `✦<style>*{background:#62306D}*+*{background:#F7EC7D;padding:168 154.4;color:62306d;font:1in'';clip-path:inset(65q 112px 226px round 9in 9in 0 0` <p> It is 143 chars long` <p> What am I missing here?

**Reply by SelenIT** [reply]
You can make the main shape with unicode characters as well, without clip-path

**Reply by joewinbpaul** [reply]
okay?... <p> I will just keep on experimenting like always..
